.github/workflows/generate-docs.yml

```diff
@@ -76,16 +79,19 @@ jobs:
           echo "*** Check for Sphinx Warnings ***"
           grep -q WARNING make.out && exit 1
           rm make.out
-          echo "*** Pushing zeek-docs Changes ***"
+
+      - name: Push zeek-docs Changes
+        if: github.event_name == 'schedule'
+        run: |
+          cd doc
           git remote set-url origin "https://zeek-bot:${{ secrets.ZEEK_BOT_TOKEN }}@github.com/zeek/zeek-docs"
           git add scripts/ script-reference/
           git status
           git commit -m "Generate docs" && git push || /bin/true
-          cd ..
```

Comment:
Does this still need to happen for the update stage to work?

Reply:
I was wondering too. Per the docs each `run` starts a new shell, so I believe no.